solution/ui/regulations/utilities/utils.js

```diff
@@ -505,18 +502,249 @@ const getCurrentPageResultsRange = ({ count, page = 1, pageSize }) => {
     return [firstInRange, lastInRange];
 };
 
+const formatResourceCategories = (resources) => {
+    const rawCategories = JSON.parse(
```

Comment:
all of these additions are the result of combining both the `eregs-vite` `utils.js` file and the `eregs-component-lib` `utils.js` file into a single utility methods file.